src/cmd/dev.go

```diff
@@ -60,6 +61,38 @@ var devDeployCmd = &cobra.Command{
 	},
 }
 
+var devGenerateCmd = &cobra.Command{
+	Use:     "generate NAME",
+	Aliases: []string{"g"},
+	Args:    cobra.ExactArgs(1),
+	Short:   lang.CmdDevGenerateShort,
+	Example: lang.CmdDevGenerateExample,
+	Run: func(_ *cobra.Command, args []string) {
+		pkgConfig.GenerateOpts.Name = args[0]
+		spinner := message.NewProgressSpinner(lang.CmdDevGenerateNewMessage, pkgConfig.GenerateOpts.Name)
+
+		// Configure and Instantiate the packager
+		pkgConfig = packager.UpdatePackageConfigForGenerate(&pkgConfig)
+		pkgClient := packager.NewOrDie(&pkgConfig)
+		defer pkgClient.ClearTempPaths()
+
+		// Update the Package with Images
+		if err := pkgClient.FindImagesWithPackage(); err != nil {
+			message.WarnErr(err, lang.CmdDevGenerateFoundNoImagesErr)
+		}
+
+		// Validate the package
+		if err := validate.Run(pkgConfig.Pkg); err != nil {
+			message.Fatalf(err, err.Error())
+		}
+
+		// Write the generated zarf.yaml
+		packager.WriteGeneratedZarfPackage(&pkgConfig)
+
+		spinner.Successf(lang.CmdDevGenerateSuccess)
```

Comment:
This spinner causes conflicts with the other spinners causing garbled output when the command finishes:
![image](https://github.com/defenseunicorns/zarf/assets/3977569/5b9ffd1a-006d-4b7c-bbe5-e04539a21032)

It would be better if this was just a `message.Success()`:
```suggestion
		message.Success(lang.CmdDevGenerateSuccess)
```